src/server/list_family.cc

```diff
@@ -565,6 +563,68 @@ void ListFamily::LLen(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void ListFamily::LPos(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 1);
+  string_view elem = ArgS(args, 2);
+
+  int rank = 1;
+  int count = -1;
+  int max_len = 0;
+
+  for (size_t i = 3; i < args.size(); i++) {
+    ToUpper(&args[i]);
+    const auto& arg_v = ArgS(args, i);
+    if (arg_v == "RANK") {
+      if (!absl::SimpleAtoi(ArgS(args, (i + 1)), &rank)) {
+        return (*cntx)->SendError(OpStatus::WRONG_TYPE);
+      } else if (rank == 0) {
+        return (*cntx)->SendError(OpStatus::INVALID_NUMERIC_RESULT);
+      }
+    }
+    if (arg_v == "COUNT") {
+      if (!absl::SimpleAtoi(ArgS(args, (i + 1)), &count)) {
```

Comment:
in case of "count" and "maxlen" - they can not be negative, so just declare them as `uint32_t` and SimpleAtoi will fail if a negative number is provided.

Reply:
also pls use `kInvalidIntErr` here as well.